tst/schedulers/test_botorch.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from examples.training_scripts.height_example.train_height import (
+    height_config_space,
+    METRIC_ATTR,
+)
+
+from syne_tune import Tuner, StoppingCriterion
+from syne_tune.backend import LocalBackend
+import pytest
+import sys
+
+
+@pytest.mark.timeout(20)
+@pytest.mark.skipif(
+    sys.version_info < (3, 8), reason="BoTorch requires python 3.8 or higher"
+)
+def test_botorch_reproducible():
+    """
+    This test checks that the BoTorch scheduler can be loaded and used to come up with new trials.
+    """
+    from syne_tune.optimizer.baselines import BoTorch
+
+    # Use train_height backend for our tests
+    entry_point = "examples/training_scripts/height_example/train_height.py"
```

Comment:
This path would break if you run from a different directory, can you use something like this to avoid that side-effect:
https://github.com/awslabs/syne-tune/blob/main/examples/launch_height_baselines.py#L39